src/librustc_mir_build/build/matches/test.rs

```diff
@@ -233,10 +230,11 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
                 } else {
                     // The switch may be inexhaustive so we have a catch all block
                     debug_assert_eq!(options.len() + 1, target_blocks.len());
+                    let options: Vec<u128> = options.values().copied().collect();
                     TerminatorKind::SwitchInt {
                         discr: Operand::Copy(place),
                         switch_ty,
-                        values: options.clone().into(),
+                        values: options.into(),
```

Comment:
Hm this looks a bit weird, is there a reason this isn't `options.clone()`? Did I miss something?

Reply:
The `values` field is a `Cow`. So before, we had a local `&Vec<u128>` that could be cloned and converted, and now we have a map and have to collect its values. I guess we could directly collect a `Cow` though -- I'll do that.